src/app/app.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterOutlet, Router, NavigationEnd } from '@angular/router';
+import { FooterComponent } from './footer/footer.component';
+import { HeaderComponent } from './header/header.component';
+import { UserSettingsService } from './user-settings.service';
+import { DataService } from './data.service';
+import { SpinnerComponent } from './spinner/spinner.component';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [CommonModule, RouterOutlet, FooterComponent, HeaderComponent, SpinnerComponent],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.css'
+})
+export class AppComponent {
```

Comment:
`AppComponent` has method `ngOnInit` but does not implement `OnInit` Interface. - Found by ESLint